packages/dapp/helpers/index.js

```diff
@@ -0,0 +1,30 @@
+import DRecruiter from '../contracts/hardhat_contracts.json'
+import { ethers } from 'ethers'
+import { abis } from "./abi";
+
+export const loadDRecruitV1Contract = async (targetNetwork, signer) => {
+  console.log('Loading DRecruit contract...')
+  console.log(`Target network: ${targetNetwork}`)
+  const contract = new ethers.Contract(
```

Comment:
Why we need this load contract function if we are loading the contract here https://github.com/moonshotcollective/recruiter.party/pull/6/files#diff-b03267768cd4f21d635cd49366ee12debcbe6ccb425a314bbd8806fe08fb612bL176 ?

Reply:
We no longer need to load the contract now as Huxwell is storing in `Web3Context`, now we just need to use it from there

Reply:
So we can remove this load functions.